src/bindings/js/node/src/addon.cpp

```diff
@@ -30,6 +30,9 @@ void init_class(Napi::Env env,
     exports.Set(class_name, prototype);
 }
 
+Prototype func = NodeWrap::get_class()
```

Comment:
Where is the implementation of `NodeWrap::get_class()`?

Reply:
whoops, it must have gotten deleted somehow when I had my git issues. I'll add it when I delete the submodules

Reply:
I'll wait :)

Reply:
Here [how to remove the issue with submodules](https://github.com/openvinotoolkit/openvino/pull/25760#pullrequestreview-2211815275)

Reply:
I'm having a bit of trouble, I have to cherry-pick the commits but it seems there's various versions of one of the commits (121288d0f3dbc94b2dc92e3da6924d0a9de63e00) and the files staged for change when I choose one include submodules. First time I'm running into this issue with git :)

Reply:
Hi @itsaflamingo, you can try to cherry-pick the faulty commit with `-n` option, which will only stage the changes, not commit them. That way you'll be able to `git add` everything you need and omit the changes to submodules.